File: iOSbeautyOfDevelop/chapter4/4.1/AddSubView/AddSubView/ViewController.m
//
//  ViewController.m
//  AddSubView
//
//  Created by aa64mac on 8/16/15.
//  Copyright (c) 2015 cc. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    CGRect labelRect = CGRectMake(120, 240, 200, 100);
    
    UILabel *label = [[UILabel alloc] initWithFrame:labelRect];
    
    label.text = @"创建的Label";
    
    [self.view addSubview:label];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: iOSDevGuide/chapter09/9.1.5/fibonacci/fibonacci/ViewController.swift
//
//  ViewController.swift
//  Fibonacci
//
//  Created by aa64mac on 10/20/15.
//  Copyright © 2015 cyj. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var theSlider: UISlider!
    @IBOutlet weak var outputTextView: UITextView!
    @IBOutlet weak var selectedValueLabel: UILabel!
    
    var fibo: FibonacciModel = FibonacciModel()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func sliderValueDidChange(sender: UISlider) {
        var returnedArray: [Int] = []
        var formattedOutput: String = ""
        
        self.selectedValueLabel!.text = String(Int(theSlider!.value))
        
        returnedArray = self.fibo.calculateFibonacciNumbers(minimum2: Int(theSlider!.value))
        for number in returnedArray {
            formattedOutput = formattedOutput + String(number) + ", "
        }
        
        self.outputTextView!.text = formattedOutput
    }

}

